src/cloud/events.py

```diff
@@ -82,7 +82,7 @@ def _send_events_loop(self):
             if not self._batch_send_events():
                 raise DaemonThreadWait
         except APIException as ex:
-            logger.error('Error sending events to the cloud %', ex)
+            logger.error('Error sending events to the cloud {}'.format(ex))
```

Comment:
Using percent formatting logging is generally best practice https://docs.python.org/2.7/library/logging.html#module-level-functions, but just % isn't a valid formatter.

```suggestion
            logger.error('Error sending events to the cloud %s', ex)
```